gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderCustomLogoExample.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+
+const customLogo = require('../../resources/logos/logo_nexus_sonatype_teal.svg');
+
+import { NxButton, NxNexusPageHeader, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
```

Comment:
We like to put external imports first
```suggestion
import { NxButton, NxNexusPageHeader, NxFontAwesomeIcon } from '@sonatype/react-shared-components';

const customLogo = require('../../resources/logos/logo_nexus_sonatype_teal.svg');
```